src/components/table.js

```diff
@@ -370,6 +370,36 @@ export class GridTable extends Component {
   }
 }
 
+export const SimpleTable = ({ columns, rows }) => {
+  const rowHeight = 24
+  const cellStyles = { display: 'flex', alignItems: 'center' }
+  return h(Fragment, [
+    div({ style: { display: 'flex', height: rowHeight } }, [
+      _.map(({ key, header, size }) => {
+        return div({ key, style: { ...cellStyles, ...styles.flexCell(size) } }, [header])
+      }, columns)
+    ]),
+    _.map(([i, row]) => {
+      return h(Interactive, {
+        key: i,
```

Comment:
Why bother to key these if you're just using their indices?

Reply:
Keyed collections help react be a little smarter about DOM re-use, which can help performance in some cases. It's probably not a big help here, but it's a good habit to get into.